docs/html/_sources/sandbox.rst.txt

```diff
@@ -1,30 +1,30 @@
-Sandbox
+Localnet
 ===========
 
-Beaker provides helper methods to work with a local `sandbox  <https://github.com/algorand/sandbox>`_.
+Beaker provides helper methods to work with a `algokit  <https://github.com/algorandfoundation/algokit-cli>`_ localnet.
```

Comment:
@joe-p Can we say 

"Beaker provides helper methods to work with an `algokit  <https://github.com/algorandfoundation/algokit-cli>`_ localnet or with a local `sandbox  <https://github.com/algorand/sandbox>`_ network.

Reply:
Done in [a5806f1](https://github.com/algorand-devrel/beaker/pull/238/commits/a5806f1f00cae57caa598a023ad956dc241d95e0)